### ISSUE TITLE
Dark mode is not completely disabled

### ISSUE THREAD

**Issue by tr4nk** [sebastienheyd/boilerplate#84]
After set `'darkmode' => false` in `config/boilerplate/theme.php`, it works except the following 2 cases:
- Login page is still in dark mode.
- Register a new user and the first time that user logs in, dark mode is still used. Then reload the page, light mode is used.

**Comment by sebastienheyd** [comment]
That's right! That's because the automatic dark mode detection is active on these pages. I'll try to fix this asap
